packages/ethereum/contracts/test/Channels.t.sol

```diff
@@ -0,0 +1,1441 @@
+      keccak256(abi.encode(channelBA))
+    );
+  }
+
+  function testSetCommitmentEpochBump(uint96 amount1, uint96 amount2) public {
+    amount1 = uint96(bound(amount1, MIN_USED_BALANCE, MAX_USED_BALANCE));
+    amount2 = uint96(bound(amount2, MIN_USED_BALANCE, MAX_USED_BALANCE));
+
+    bytes32 commitment1 = 0xa74616c1018b7fccab7c6d508560a93768df100a80661bf2693d79d5b474d75e;
+    bytes32 commitment2 = 0x677cbfec2c3998a7ed96e60b2f408abe400313dc5f365a465ab6d40888b18375;
+
+    HoprChannels.Channel memory channelAB = HoprChannels.Channel(
+      bytes32(commitment1),
+      HoprChannels.Balance.wrap(uint96(amount1)),
+      HoprChannels.TicketIndex.wrap(0),
+      HoprChannels.Timestamp.wrap(0),
+
+      HoprChannels.ChannelEpoch.wrap(1),
+      HoprChannels.ChannelStatus.OPEN
+    );
+
+    vm.prank(address(1));
+    vm.mockCall(
+      vm.addr(1),
+      abi.encodeWithSignature(
+        'transferFrom(address,address,uint256)',
+        address(1),
+        address(hoprChannels),
+        amount1 + amount2
+      ),
+      abi.encode(true)
+    );
+
+    vm.expectEmit(true, false, false, false, address(hoprChannels));
+    emit ChannelOpened(accountA.accountAddr, accountB.accountAddr, HoprChannels.Balance.wrap(amount1));
+
+    vm.expectEmit(true, false, false, false, address(hoprChannels));
+    emit ChannelOpened(accountB.accountAddr, accountA.accountAddr, HoprChannels.Balance.wrap(amount2));
+    
+    hoprChannels.fundChannelMulti(accountA.accountAddr, HoprChannels.Balance.wrap(amount1), accountB.accountAddr, HoprChannels.Balance.wrap(amount2));
+    vm.clearMockedCalls();
+
+    vm.expectEmit(true, false, false, true, address(hoprChannels));
+    emit CommitmentSet(channelIdAB, HoprChannels.ChannelEpoch.wrap(1));
+
+    vm.prank(accountB.accountAddr);
+    hoprChannels.setCommitment(bytes32(commitment1), accountA.accountAddr);
+
+    assertEq(
+      keccak256(
+        abi.encode(getChannelFromTuple(channelIdAB))
+      ),
+      keccak256(abi.encode(channelAB))
+    );
+
+    vm.prank(accountB.accountAddr);
+    vm.expectRevert(abi.encodeWithSelector(InvalidCommitment.selector));
+    hoprChannels.setCommitment(bytes32(commitment1), accountA.accountAddr);
+
+    HoprChannels.Channel memory channelABBumped = HoprChannels.Channel(
+      bytes32(commitment2),
+      HoprChannels.Balance.wrap(uint96(amount1)),
+      HoprChannels.TicketIndex.wrap(0),
+      HoprChannels.Timestamp.wrap(0),
+
+      HoprChannels.ChannelEpoch.wrap(2),
+      HoprChannels.ChannelStatus.OPEN
+    );
+
+
+    vm.expectEmit(true, false, false, true, address(hoprChannels));
+    emit CommitmentSet(channelIdAB, HoprChannels.ChannelEpoch.wrap(2));
+
+    vm.prank(accountB.accountAddr);
+    hoprChannels.setCommitment(bytes32(commitment2), accountA.accountAddr);
+
+    assertEq(
+      keccak256(
+        abi.encode(getChannelFromTuple(channelIdAB))
+      ),
+      keccak256(abi.encode(channelABBumped))
+    );
+  }
+
+  function testFailSetCommitment() public {
```

Comment:
Similarly as above, when a function is reverted with a specific error message, use `testRevert` prefix.

Reply:
thx, changed